odoo/addons/base/rng/common.rng

```diff
@@ -267,6 +267,7 @@
             <rng:optional><rng:attribute name="write_field" /></rng:optional>
             <rng:optional><rng:attribute name="text" /></rng:optional>
             <rng:optional><rng:attribute name="optional" /></rng:optional>
+            <rng:optional><rng:attribute name="create_text"/></rng:optional>
```

Comment:
`create_text` is unclear, this looks like a label for the "Add" button to me.
I know the option already existed, but it's time to change it if you make it part of the RNG ;-)
So maybe `add-label`? Also note that the majority of attributes are using hyphens as separators, let's aim for more consistency, not less.

Reply:
@odony 👍